posthog/tasks/status_report.py

```diff
@@ -69,7 +69,7 @@ def status_report(*, dry_run: bool = False) -> Dict[str, Any]:
             params = (team.id, report["period"]["start_inclusive"], report["period"]["end_inclusive"])
             team_report: Dict[str, Any] = {}
             # pull events stats from clickhouse
-            from ee.clickhouse.models.event import (
+            from posthog.models.event.util import (
```

Comment:
Hmm, some of these imports are at the top, others here. Worth moving all to the top now? (including the `posthog.models.person` ones right below